netjson-monitoring.lua

```diff
@@ -72,6 +72,24 @@ netjson = {
     }
 }
 
+-- collect interface addresses
+interface_data = ubus:call('network.interface', 'dump', {})
+addresses = {}
+
+function get_address()
+  for _, interface_list in pairs(interface_data) do
+    for _, interface in pairs(interface_list) do
+      if interface["ipv4-address"] then
+        table.insert(addresses, interface["ipv4-address"][1])
```

Comment:
`interface["ipv4-address"]` is a list, so you need to loop over the it because there might be multiple ips associated to the interface